### PR TITLE
command-line option to ignore non-run functions

### PR DESCRIPTION
**Pull request by ta946**
add param -k, --skip to skip displaying functions that have not been run and have 0 runtime

### CODE REVIEW

**Comment by ta946** [comment]
im unfamiliar with github's PRs and so it seems resolving a merge conflict on my fork ended up affecting this PR and is now mixed with the code from another commit.
the real commit for this PR should be ea841173cfda71326e563218cde85f368286230e, before the merge.

File: line_profiler/line_profiler.py
Comment (Erotemic) [inline line_profiler/line_profiler.py:445]: Please change: 

```python
skip = True if options.skip else False
```
image

----

to: 

```python
skip = options.skip
```
image

----

or 

```python
skip = bool(options.skip)
``` 

to have exact equivalence to the original code, but the value of `options.skip` should always be a bool, so there is no need for that. Having the if-else statement is superfluous.

**Review by Erotemic — CHANGES_REQUESTED** [review]
Pending the small requested change, this LGTM. Once #22 is merged, we can rebase and then merge this.

File: line_profiler/line_profiler.py
Comment (ta946) [reply line_profiler/line_profiler.py:445]: ill use bool(options.skip) because it needs to parse None as False aswell

**Comment by artemisart** [comment]
@ta946 @Erotemic this could be simplified as `stripzeros` already does exactly that and you just need to pass a command line argument. Should I make a PR for that or can you just update this one ?
Also I would like to update the code to use argparse (which is really nicer and simplify the code), should I wait for this PR and #22 to be merged before ?

**Comment by ta946** [comment]
@artemisart you're right, just double checked and its exactly the same 😅
i would suggest creating a new PR as this one is abit of a mess.
I also suggest using "-s" instead of the argument "k" that I used because it seems "-s" is used for the line_magic argument, so better to keep it consistent. Idk if you want to use "skip" which is the expanded argument name or "strip"/"stripzeros" which is what is written in the line magic comments, but i guess i missed stripzeros exists bc the name wasnt intuitive.

If you are going to change it to argparse, id say wait till #22 is merged as it would then be easier to merge without conflicts if you can change it all together

**Comment by Erotemic** [comment]
Closed in favor of #34

**Pull request closed without merging**
